pkgs/servers/mir/default.nix

```diff
@@ -0,0 +1,126 @@
+{ lib
+, pkgs
+, stdenv
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, boost
+, egl-wayland
+, libglvnd
+, glm
+, glog
+, lttng-ust
+, udev
+, glib
+, wayland
+, libxcb
+, xorg
+, libdrm
+, mesa
+, libepoxy
+, nettle
+, libxkbcommon
+, libinput
+, libxmlxx
+, libuuid
+, pcre2
+, freetype
+, libyamlcpp
+, python3Packages
+, libevdev
+, libselinux
+, libsepol
+, pcre
+, eglexternalplatform
+, doxygen
+, libxslt
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mir";
+  version = "2.10.0";
```

Comment:
Oh you're right, good point https://github.com/MirServer/mir/issues/1744

I guess we'll have to add Mir 1.X with qtmir in a separate PR. Since Mir can be used outside of qtmir, we would start here with package the most recent version